packages/address/src/AddressFormatter.ts

```diff
@@ -17,19 +23,18 @@ export default class AddressFormatter {
   }
 
   async getCountry(countryCode: string): Promise<Country> {
-    let country = this.loadCountryFromCache(countryCode);
-    if (country) {
-      return country;
-    }
-
-    country = await loadCountry(this.locale, countryCode);
-
-    return country;
+    return Promise.resolve(
+      this.loadCountryFromCache(countryCode) ||
+        loadCountry(this.locale, countryCode),
+    );
   }
 
   async getCountries(): Promise<Country[]> {
+    const cachedCountries = ORDERED_COUNTRIES_CACHE.get(this.locale);
```

Comment:
Good point!

Unless the list of countries we are serving through the atlas service ever change, it could prevent a full refetch in a single session, though a page refresh (or the next browser session) would get the updated list.

I personally think it was an oversight, and now that we have a `resetCache` method, any frontend that wants to force a refetch of the list could do it, but I don't believe it's a real use-case. 

Noting it in the changelog is a good idea, if you're up with keeping this change.